### PR TITLE
Update CONTRIBUTING with Intellij instructions

### LINKED ISSUE #1479: Document how to set up project in IntelliJ
While we have documentation that helps contributors set up and run the project from the command line, a large number of contributors also use IDEs to make changes to files.

IntelliJ seems to be the most popular IDE that is used (though others are mentioned [here](https://github.com/exercism/java/blob/master/docs/INSTALLATION.md), so we should update the [CONTRIBUTING file](https://github.com/exercism/java/blob/master/CONTRIBUTING.md) to include instructions to pull in the project.

### PR DESCRIPTION
**Pull request by delbertlegg**
Updated CONTRIBUTING to include instructions for how to import into Intellij and use VCS tools.

Closes #1479 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

**Comment by delbertlegg** [comment]
Sure thing, updated! 🚀

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,21 +77,39 @@ like to clone the repo into.
...
+ ![diff](assets/run-test.png)
+
  - **Commit/Merge changes**  Once all of the changes have been made, you can look at the diffs and commit from the "Commit File" window, which can be reached by selecting VCS > Git > Commit File from the top menu.
  If all of the changes are acceptable, checkmark all of the files that are to be committed, enter a meaningful commit message, and then click "Commit and Push".    
  
  ![diff](assets/commit-files.png)
  
  - Follow the instructions regarding creating a pull request into the upstream repo.
  
+ **NOTE:** Git and gradle commands can still be ran in the command line when using and IDE. The steps outlining how to perform using IDE tools are for convenience only.
```
Comment (FridaTveit) [inline CONTRIBUTING.md]: Small typo, it should be "can still be run" I believe 🙂

Comment (delbertlegg) [reply CONTRIBUTING.md]: Sure thing, done 👍

**Comment by delbertlegg** [comment]
Happy to help! Looking for more tasks if there are any.

**Pull request merged**